Formula/p/pnpm@9.rb

```diff
@@ -0,0 +1,53 @@
+class PnpmAT9 < Formula
+  desc "Fast, disk space efficient package manager"
+  homepage "https://pnpm.io/"
+  url "https://registry.npmjs.org/pnpm/-/pnpm-9.15.3.tgz"
+  sha256 "c1da43727ccbc1ed42aff4fd6bdb4b1e91e65a818e6efff5b240fbf070ba4eaf"
+  license "MIT"
+
+  livecheck do
+    url "https://registry.npmjs.org/pnpm/latest-9"
+    regex(/["']version["']:\s*?["']([^"']+)["']/i)
+  end
+
+  bottle do
+    sha256 cellar: :any,                 arm64_sequoia: "e51e36734f36217f5e88edf34d05b76187e95b2a9a04110455e6a39f1c094515"
+    sha256 cellar: :any,                 arm64_sonoma:  "e51e36734f36217f5e88edf34d05b76187e95b2a9a04110455e6a39f1c094515"
+    sha256 cellar: :any,                 arm64_ventura: "e51e36734f36217f5e88edf34d05b76187e95b2a9a04110455e6a39f1c094515"
+    sha256 cellar: :any,                 sonoma:        "3466d76ac687a7d374e8606621bcd0848b9248a43a2c98f57699a0a8d2c3db12"
+    sha256 cellar: :any,                 ventura:       "3466d76ac687a7d374e8606621bcd0848b9248a43a2c98f57699a0a8d2c3db12"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:  "d422afed7a35b78c0fb5f538e2ec4bd09166dfd81d4d2a541b51abafee833757"
+  end
+
```

Comment:
```suggestion
```

Reply:
I hope it will be updated by build process, or should I make commit?